### PR TITLE
Increase versions of base nodejs images to match most recent versions.

### PR DESCRIPTION
**Pull request by geodn**
## Description
This change updates the nodejs versions in our base images to be the most recent versions.

## Test plan
docker pull

### CODE REVIEW

File: pkg/build/versions.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "node": {
     "18": {
       "image": "registry.hub.docker.com/library/node",
-      "tag": "18.1.0-buster",
-      "digest": "sha256:3b88dc7e2379f5adb2e2893e0d4cf1774e260c176e51a6a8fc0cb358a2749a3f"
+      "tag": "18.7.0-buster",
```
Comment (colinking) [inline pkg/build/versions.json:5]: Debian 10 (buster) has been replaced by Debian 11 (bullseye). Let's upgrade that.

File: pkg/build/versions.json [hunk 2]
```diff
@@ -17,8 +17,8 @@
     },
     "14": {
       "image": "registry.hub.docker.com/library/node",
-      "tag": "14.16.1-buster",
-      "digest": "sha256:3b325b40048d329876134243db80718413b1410d5371a94823b3e4e5b71721b2"
+      "tag": "14.20.0-buster",
+      "digest": "sha256:3e7c9e5fe90474a3e9fbd88c59b32a9f0fc2dc34cc2ffb1c0c418956e25d9334"
     },
     "12": {
```
Comment (colinking) [inline pkg/build/versions.json:23]: Let's upgrade to 12.22.12-bullseye

**Review by colinking — APPROVED** [review]
Let's also upgrade Python to 3.10.6-bullseye. Thanks!

Once you're ready to merge, make sure to push these digests into our public cache, do a release of `lib`, and merge that into CLI and airport. LMK if you have any questions doing that.

File: pkg/build/versions.json (hunk 2 above)
Comment (geodn) [reply pkg/build/versions.json:23]: Done.

File: pkg/build/versions.json (hunk 1 above)
Comment (geodn) [reply pkg/build/versions.json]: Done.

**Pull request merged**